src/shell/interpreter.zig

```diff
@@ -2152,6 +2151,37 @@ pub const Interpreter = struct {
         }
 
         fn expandVar(this: *const Expansion, label: []const u8) EnvStr {
+            if (bun.strings.isAllASCIIDigit(label)) {
+                var int = std.fmt.parseInt(u32, label, 10) catch return EnvStr.initSlice("");
+                switch (this.base.interpreter.event_loop) {
+                    .js => |js| {
+                        if (int == 0) return EnvStr.initSlice(bun.selfExePath() catch "");
+                        int -= 1;
+
+                        const vm = js.virtual_machine;
+                        if (vm.main.len > 0) {
+                            if (int == 0) return EnvStr.initSlice(vm.main);
+                            int -= 1;
+                        }
+
+                        if (vm.worker) |worker| {
+                            if (worker.argv) |argv| {
+                                if (int >= argv.len) return EnvStr.initSlice("");
+                                return EnvStr.initSlice(argv[int].utf8Slice());
```

Comment:
Are we sure `argv[int]` will always be utf8?

Reply:
not sure, I can try some weirder examples. my first guess would be I dont think so because they're not coming JS